cmake/options.cmake

```diff
@@ -74,5 +74,9 @@ endif (WITH_ZEPHYR)
 
 option (WITH_LIBMETAL_FIND "Check Libmetal library can be found" ON)
 
+if (DEFINED RPMSG_BUFFER_SIZE)
+  add_definitions( -DRPMSG_BUFFER_SIZE=${RPMSG_BUFFER_SIZE} )
```

Comment:
This needs a documentation update somewhere to tell the user that they can change the default, and that changing the default will make it incompatible with the Linux kernel implementation. I think that this change or #155 is major enough to bring up in the next openamp-rp call, or on the mailing list.

Reply:
Hi @edmooring,
Agree on the principle for the Documentation. We have to discuss the way to provide a documentation accessible to contributors.
Concerning this commit itself, it is not a major change. The RPMSG_BUFFER_SIZE can already be customized (https://github.com/OpenAMP/open-amp/blob/master/lib/include/openamp/rpmsg_virtio.h#L25)
So this patch only exports the definition, to be able to define the size in the make command line instead of updating the makefiles.

Reply:
Hi @arnopo 
For now, putting it in README.md, near the instructions for building OpenAMP for the various platforms would be fine with me.

Reply:
That makes sense, as some other cmake options have also to be defined (e.g WITH_VIRTIO_SLAVE, WITH_VIRTIO_MASTER ) , i would propose to have a dedicated thread for this.